extensions/wikia/VideoHandlers/apiwrappers/YoutubeApiWrapper.class.php

```diff
@@ -101,8 +109,6 @@ protected function getVideoTitle() {
 		if ( !empty( $this->interfaceObj['snippet']['title'] ) ) {
 			return $this->interfaceObj['snippet']['title'];
 		}
-
-		return '';
```

Comment:
It should return something (consistency!). If not an empty string then null at least.